armi/reactor/blueprints/reactorBlueprint.py

```diff
@@ -116,12 +167,11 @@ def _loadAssemblies(self, cs, container, gridDesign, gridContents, bp):
 
             i, j = locationInfo
             loc = container.spatialGrid[i, j, 0]
-            if (
-                container.symmetry == geometry.THIRD_CORE + geometry.PERIODIC
-                and not container.spatialGrid.isInFirstThird(loc, includeTopEdge=True)
-            ):
+            try:
+                container.add(newAssembly, loc)
+            except exceptions.SymmetryError:
```

Comment:
will this  handle geoms that had the extra edge assemblies differently? If so we should add to list of changes that will eventually become release notes

Reply:
No, I had considered making this more strict, but decided to make the symmetry line part of the domain, so the behavior should be unchanged. The issue comes down to edge assems specifically; since they dont fill their respective grid locations, grid locations arent as useful as a determination of whether something is or isnt in the domain. we may need to make the idea of shared edges somewhat explicit to support split assems

Reply:
Alright, i bubbled the idea of `symmetryOverlap` up to the `Grid`; let me know what you think